codec-http/src/main/java/io/netty/handler/codec/http/cookie/ClientCookieDecoder.java

```diff
@@ -168,7 +168,7 @@ private long mergeMaxAgeAndExpires() {
             if (maxAge != Long.MIN_VALUE) {
                 return maxAge;
             } else if (isValueDefined(expiresStart, expiresEnd)) {
-                Date expiresDate = HttpHeaderDateFormatter.parse(header, expiresStart, expiresEnd);
+                Date expiresDate = HeaderDateFormatter.parse(header, expiresStart, expiresEnd);
```

Comment:
Here, I just rename the class.
Before previous PR, exceptions were trapped/silent as malformed date should be considered as a session cookie.